src/include/mapping.h

```diff
@@ -50,6 +50,7 @@
 #define MAPPING_MAYSHARE	0x100000
 #define MAPPING_NOOVERLAP	0x200000
 #define MAPPING_COPYBACK	0x400000
+#define MAPPING_LOG_DIRTY	0x800000
```

Comment:
Hmm, this looks too kvm-specific.
How about just special-casing MAPPING_VGAEMU
in KVM code?